tests/Knp/Menu/Tests/Provider/ChainProviderTest.php

```diff
@@ -6,9 +6,12 @@
 use Knp\Menu\Provider\ChainProvider;
 use Knp\Menu\Provider\MenuProviderInterface;
 use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
```

Comment:
you need to add the dev dependency, otherwise the trait does not exist

Reply:
Added in https://github.com/KnpLabs/KnpMenu/pull/352/commits/73b0cdf89e3043527af6aa148320e7e981491a42